swf/src/write.rs

```diff
@@ -2244,6 +2173,44 @@ impl<W: Write> Writer<W> {
         Ok(())
     }
 
+    fn write_define_font_info(&mut self, font_info: &FontInfo) -> Result<()> {
+        let use_wide_codes = self.version >= 6 || font_info.version >= 2;
```

Comment:
This is now specified in `self.flags`. We might want to just use `self.flags` instead of this version check.